README.md

```diff
@@ -530,6 +530,8 @@ If you're using ProGuard, you may need to add rules to your configuration file t
 # Core SDK
 -keep class com.sdk.growthbook.** { *; }
 
+-keep class kotlinx.serialization.** { *; }
```

Comment:
This rule can be even more concrete:
```
-keep class kotlinx.serialization.json.** { *; }
```

Reply:
Merging this in main will fail because library version is not increased.
In order to merge this change we created [new pull request](https://github.com/growthbook/growthbook-kotlin/pull/64).

Reply:
This change was merged in [other pull request](https://github.com/growthbook/growthbook-kotlin/pull/64), so we can close this